src/Workspaces/CSharp/Portable/Extensions/ExpressionSyntaxExtensions.cs

```diff
@@ -2578,6 +2578,11 @@ public static SimpleNameSyntax GetRightmostName(this ExpressionSyntax node)
                 return memberBinding.Name;
             }
 
+            if (node is AliasQualifiedNameSyntax aliasQualifiedName && aliasQualifiedName.Name != null)
+            {
+                return aliasQualifiedName.Name;
```

Comment:
If this fixes a crash, then it seems like that would be because returning null below causes a problem.  Which means the caller of this method shoul dbe updated to be resilient to getting null back, as that's part of the contract of this method.

Reply:
Good point, thanks!

Reply:
@zaytsev-victor Should we undo this change then, since the change in NameSyntaxClassifier.cs is the right one?

Reply:
@jasonmalinowski No, we should not undo this change, because this the fix. Without this change
```Col::List``` would not be recognized that ```List``` is the type.

Reply:
Consider renamign this method to TryGetRightmostName.  That will allow us to audit all the callers.